subdomains/docs/_guide/understanding.md

```diff
@@ -13,8 +13,7 @@ Similar to package managers, Volta keeps track of which project (if any) you're
 You control the tools managed by your Volta toolchain with two commands: `volta install` and `volta uninstall`.
 
 ### Installing Node engines
-
-When you install a tool to your toolchain, you always choose a _default version_ of that tool, which Volta will use unless you're working within a project directory that has configured Volta to use a different version.
+Every time you install a tool to your toolchain, you set the default version of that tool, which Volta will use unless you're working within a project directory that has configured Volta to use a different version.
 
 For example, you can select your default version of `node` by installing a particular version:
```

Comment:
With the above suggestion, could we also cut out the reference to "installing" here? Perhaps something like:

```suggestion
For example, you can select an exact version of `node` to be your default version:
```

Reply:
Yes I like these changes! In most apps/tools "when you install" usually means the first time,  so emphasizing that the command is primarily meant to set the default version clears things up. I incorporated your suggestions with a minor change of splitting a long winded sentence. Let me know if you have any additional feedback. And thanks for making Volta, it's great!